From e9add5f066cb0209ff9f6a0ef71584b54afd04d7 Mon Sep 17 00:00:00 2001
Message-Id: <e9add5f066cb0209ff9f6a0ef71584b54afd04d7.1610665323.git.liu.denton@gmail.com>
In-Reply-To: <cover.1610665323.git.liu.denton@gmail.com>
References: <20210112201806.13284-1-avarab@gmail.com>
	<cover.1610665323.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Thu, 14 Jan 2021 12:21:11 -0800
Subject: [PATCH 1/2] test-lib-functions.sh: fix usage for test_commit()
To: Git Mailing List <git@vger.kernel.org>
Cc: Junio C Hamano <gitster@pobox.com>,
    Johannes Sixt <j6t@kdbg.org>,
    Johannes Schindelin <Johannes.Schindelin@gmx.de>,
    brian m . carlson <sandals@crustytoothpaste.net>,
    Ævar Arnfjörð Bjarmason <avarab@gmail.com>

The usage comment for test_commit() shows that the --author option
should be given as `--author=<author>`. However, this is incorrect as it
only works when given as `--author <author>`. Correct this erroneous
text.

Also, for the sake of correctness, fix the description as well since we
invoke `git commit` with `--author <author>`, not `--author=<author>`.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 t/test-lib-functions.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/test-lib-functions.sh b/t/test-lib-functions.sh
index b0a5d74dc7..cea73cb8f0 100644
--- a/t/test-lib-functions.sh
+++ b/t/test-lib-functions.sh
@@ -188,8 +188,8 @@ debug () {
 #	"<file>"
 #   --signoff
 #	Invoke "git commit" with --signoff
-#   --author=<author>
-#	Invoke "git commit" with --author=<author>
+#   --author <author>
+#	Invoke "git commit" with --author <author>
 #
 # This will commit a file with the given contents and the given commit
 # message, and tag the resulting commit with the given tag name.
-- 
2.30.0.284.gd98b1dd5ea

